kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/AdaptiveBigQueryWriter.java

```diff
@@ -69,6 +77,12 @@ private boolean isTableMissingSchema(BigQueryException exception) {
     return exception.getReason() != null && exception.getReason().equalsIgnoreCase("invalid");
   }
 
+  private boolean isTableNotExisted(BigQueryException exception) {
```

Comment:
nit: suggest `tableNotExists()`

Reply:
Since the function is to determine if an exception is caused by table not existed, I updated the name to be `isTableNotExistedException`~

Reply:
Basically, I just want to say "Is table not existed" is not a correct sentence.